src/marketplace/MakeAComponentSaleView.js

```diff
@@ -44,50 +46,57 @@ class MakeAComponentSaleView extends React.Component<Props, State> {
 		//set the users id
 		this.setState({userId:jsonObjectOfUser._id});
 		for (let key in jsonObjectOfUser.inventory.tankComponents) {
-            //This is how many of the current item the user has
-            let amountOfItemsUserHas = jsonObjectOfUser.inventory.tankComponents[key];
-            if(amountOfItemsUserHas > 0)
-            {
-                let obj = {};
-                obj['value'] = key;
-                obj['label'] = key;
-                itemsToSell.push(obj);
-            }		
+		//This is how many of the current item the user has
+			let amountOfItemsUserHas = jsonObjectOfUser.inventory.tankComponents[key];
+			if(amountOfItemsUserHas > 0)
+			{
+				let obj = {};
+				obj['value'] = key;
+				obj['label'] = key;
+				itemsToSell.push(obj);
+			}		
 		}
 		//Need this to deal with the asynch nature of api calling......fun times
 		this.forceUpdate();
```

Comment:
uh oh. Foul play is afoot...

Reply:
Yeah this is just a toasty review these changes are coming when I handle the infinite request which I grouped together with issue 294

Reply:
cool